develop/tutorials/articles/270-forms/01-form-field-types/03-adding-form-field-type-settings.markdown

```diff
@@ -123,45 +151,48 @@ the field. Form field rules are configured using the `@DDMFormRule` annotation.
 
 The interface extends `DefaultDDMFormFieldTypeSettings`. That's why the default
 settings can be used in the class annotation without setting them up in the
-class, as was necessary for the mask and placeholder.
+class, as was necessary for the predefinedValue, min and max.
 
 | **DDM Annotations:** The `@DDMForm` annotation on this class allows the form engine to
 | convert the interface definition into a dynamic form. This makes it really
 | intuitive to lay out your settings form.
-| 
+|
 | For now, here are brief explanations for the annotations used in the above
 | example:
-| 
+|
 | `@DDMForm`
 | : Instantiates a new `DDMForm`. Creates a dynamic form from the annotation.
-| 
+|
+`@DDMFormRule`
```

Comment:
Hi @natocesarrego, I need help to explain this piece of code:

```
@DDMForm(
       rules = {
           @DDMFormRule(
               actions = {
                   "setVisible('validation', false)"
               },
               condition = "TRUE"
           )
       }
   )
```

Reply:
Yes, I can, @alinedoleron. This piece of code is responsible for create a Form Rule (@DDMFormRule) programatically. This Rule will always execute because its condition is equals to **TRUE** and the action associate to it is hide the validation of a specific Form field, in this case the Slider. Is this clear to you?

Reply:
Thank you, very much, @natocesarrego!